Source/BoundaryConditions/PML.cpp

```diff
@@ -664,6 +664,8 @@ PML::PML (const int lev, const BoxArray& grid_ba,
         auto ngFFT = IntVect(ngFFt_x, ngFFt_z);
 #elif defined(WARPX_DIM_1D_Z)
         auto ngFFT = IntVect(ngFFt_z);
+#elif defined(WARPX_DIM_RCYLINDER) || defined(WARPX_DIM_RSPHERE)
+        auto ngFFT = IntVect(ngFFt_x);
 #endif
```

Comment:
Just double checking, does `ngFFT` here need to be set for internal consistency even though the FFT-based solver is not supported in the new geometries?

Reply:
`ngFFT` is only set here to allow the code to compile. The aim was to have the changes as small as possible since this is already a large PR. The alternative is to wrap all of the PML code in macros so it it skipped when using the 1d versions, but that is more complicated that this change. Note that PML are not supported here.